src/main/java/frc/robot/commands/vision/TestVision.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.commands.vision;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.util.Vision;
+
+public class TestVision extends CommandBase{
+  private Drivetrain m_drive;
+  private Vision m_vision;
+  private Translation2d m_visionStartTranslation, m_driveStartTranslation;
+  private Pose2d m_currentPose = null;
+  private int m_endCounter = 0;
+  private int m_printCounter=0;
+  private double m_speed;
+
+  //How many frames it has to not see anything to end the command
+  private static final int endDelay = 5;
```

Comment:
inaccurate comment, this is how many frames it has not seen any thing but how many rio cycles it has acompished. If the rio runtime and april tag update frequencies were the same it wouldn't be a problem but becuse they are diffrent this is problematic.